build_tools/benchmarks/suites/cmake_rule_generator.py

```diff
@@ -18,22 +18,22 @@
 from .definitions import common_definitions, iree_definitions
 from . import iree_benchmarks
 
-TEMPLATE_DIR = pathlib.Path(__file__).parent
-
 
-def read_template_from_file(template_name: str) -> string.Template:
-  with open(TEMPLATE_DIR / template_name, "r") as f:
+def read_template_from_file(template_dir: pathlib.Path,
+                            template_name: str) -> string.Template:
+  with open(template_dir / template_name, "r") as f:
```

Comment:
It turns out that pathlib actually has a function to open, read, and close: https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text

Reply:
Replaced the open and read. I keep `read_template_from_file` since it still help convert the text into `string.Template`.